entity-registry/src/test/java/com/linkedin/metadata/models/OpenApiSpecBuilderTest.java

```diff
@@ -108,9 +108,9 @@ public void testOpenApiSpecBuilder() throws Exception {
         Path.of(getClass().getResource("/").getPath(), "open-api.yaml"),
         openapiYaml.getBytes(StandardCharsets.UTF_8));
 
-    assertEquals(openAPI.getComponents().getSchemas().size(), 914);
-    assertEquals(openAPI.getComponents().getParameters().size(), 56);
-    assertEquals(openAPI.getPaths().size(), 102);
+    assertTrue(openAPI.getComponents().getSchemas().size() >= 882);
```

Comment:
This seems like an odd spot to have varying numbers, I think we'd want to know if new schemas are showing up? Maybe it's not important though if this will change often.

Reply:
They would change on every data model update including changes to the entity-registry. The numbers are going to increase fairly constantly unless we start deprecating things. A simple non-zero check would however not be as useful since it might actually be broken for large chunks of the data models so this is trying to detect a regression rather then ensuring perfect accuracy.